.github/workflows/docker.yml

```diff
@@ -119,6 +120,6 @@ jobs:
             --tag $DOCKER_REPO:latest \
             --build-arg=BASE_IMAGE=rust:$RUST_IMAGE_TAG \
             --build-arg=CHEF_TAG=$CHEF_PACKAGE_VERSION \
-            --platform linux/amd64,linux/arm64 \
+            --platform linux/amd64,linux/arm64,linux/386,linux/arm/v7 \
```

Comment:
Don't we have the same problem here with respect to Alpine?

Reply:
latest should always be Debian, isn’t it?